tony-history-server/test/utils/TestHdfsUtils.java

```diff
@@ -0,0 +1,211 @@
+package utils;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
```

Comment:
Why are you using Mockito at all in these tests? You should be able to set up some test folders and files in a `@BeforeClass` method and then all the tests can run against those real folders/files. Using real files will make the tests easier to reason about and exercise the code more thoroughly.

Reply:
Ok I just talked to @oliverhu offline, I'll update the tests soon.

Reply:
@erwa Fixed: https://github.com/linkedin/TonY/pull/62/commits/5080c611e482d8adbf9df48941573ef69ee0f4c2